capstone/src/main/java/com/google/sps/data/Room.java

```diff
@@ -4,15 +4,15 @@
 import com.google.appengine.api.datastore.Entity;
 import java.util.ArrayList;
 import java.util.List;
-import com.google.sps.service.DatabaseService;
 
 public class Room {
     public static final String ROOM_ENTITY_NAME = "Room";
     public static final String TITLE_PROPERTY_KEY = "title";
     public static final String DESCRIPTION_PROPERTY_KEY = "description";
     public static final String HOST_PROPERTY_KEY = "host";
     public static final String FOLLOWERS_PROPERTY_KEY = "followers";
-    
+    public static final String TAG_LIST_PROPERTY_KEY = "tagList";
```

Comment:
I think TAGS_PROPERTY_KEY may be better here

Reply:
Yup